### PR TITLE
Upstream sync to Fork guide

### PR DESCRIPTION
**Pull request by Rusty-Gravy**
Just a simplified .txt guide on how to pull (push???) and manage commits from
Stuicey/ApolloStation to LoserThatForked/Apollostation

I struggled to figure out how to do this for a while, so I did some
digging and creating this guide to help newbies.

### CODE REVIEW

**Comment by nullbear** [comment]
Are you pointing fingers?

**Comment by Rusty-Gravy** [comment]
I asked Gutsy and he wasn't too sure himself on how to help me, so I made a simple guide after finding out myself.
Also I didn't mean to throw that git ignore in there, apologies.

File: HOW TO SYNC TO YOUR FORK.txt [hunk 1]
```diff
@@ -0,0 +1,81 @@
+=========================== Upstream to Fork Sync through Git Shell ===========================
+===============================================================================================
+
+This is a small guide designed to help those that have forked stuicey/ApolloStation and have
```
Comment (Rusty-Gravy) [inline HOW TO SYNC TO YOUR FORK.txt]: Intro

**Comment by GutsySS13** [comment]
Most of the stuff I do (probably badly) works like this:

fork apollo locally.
do stuff to your new local project
----some time passes and apollo has updated and you think you're ready to merge----

re-fork apollo locally (you could probably merge master straight into your local if you wanted)
open the merge view window and merge the new fork into your project just to make sure every thing is compatible.

<<<<<< TEST !!!!! >>>>>>

publish your local fork to github
setup your pull request on github.

---

There is an issue when working on map files for example and you're doing lots and lots of testing as they can generate a lot of spam and map files history files that aren't really necessary so what I did in the case of Valen's map for example was publish it to github (this let me extract the final map file) roll back all the map updates and then just plonk in the final map. That way I just had 3 little change logs and all my testing and rollbacks were just in my local repository and all that got added to the apollo was just 3 little updates.

It all comes down to what you're doing though, some map updates can change half the file others just 1 line (see my recent sec proposed update that just changes 1 line). Programming updates tend to be smaller as coding just tends to be dealing with text files.

---

**Comment by Rusty-Gravy** [comment]
If that method works better for you then go for it, it sounds good because you can test the new commits locally if needed.
I use the method I found because my internet is under 1MBp/s, so it can be a pain to wait for it to download anything extra.

**Comment by GutsySS13** [comment]
For me the last re-fork and merge is just so I know I can test all my stuff with the latest updates, but i'm on a fast connection. It may not be necessary and if as you say you have a slow connection then maybe it's best not to do that. Depending on how long the project has taken I guess determines the likelihood of risk that there's going to be a problem. If you're working on the main apollo-1.dmm where there's constant tweaks and fixes going on then there may be issues.

**Comment by Rusty-Gravy** [comment]
I need to make another pull request, so I'll just keep the .txt file if it's needed.

**Pull request closed without merging**